proc/test.yaml

```diff
@@ -3,3 +3,7 @@ test:
         - name: test_thread_rand
           exec: test_thread_rand
           type: unit
+        - name: test_priority
+          harness: test_priority.py
+          targets:
+            exclude: [armv7m7-imxrt106x, armv7m7-imxrt117x]
```

Comment:
We don't need to exclude targets, this test should work on all targets

Reply:
Oh, sorry I didn't see that `alarm()` wasn't used. I will enable it, but have to wait for https://github.com/phoenix-rtos/phoenix-rtos-tests/pull/55 merge (it fixes `psh.assert_exec()` on IMXRT platforms).